### PR TITLE
Add Trino 383 release notes

### LINKED ISSUE #12556: Release notes for 383

### PR DESCRIPTION
**Pull request by colebow**
## Description

Assemble the release notes for the upcoming Trino 381 release.

> Is this change a fix, improvement, new feature, refactoring, or other?

Improvement

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Docs only.

> How would you describe this change to a non-technical end user or system administrator?

Add the release notes for Trino 381 with sections for general, security, connectors and other topics as included in the release.

## Related issues, pull requests, and links

* Fixes #12556

## Documentation

( ) No documentation is needed.
(✅) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(✅) No release notes entries required.
( ) Release notes entries required with the following suggested text:

## Verification for each commit

Format: commit, committer/maintainer, PR/issue number, ✅ / ❌ rn ✅ / ❌ docs
✅ rn - release note added and verified, or assessed to be not necessary, set to ❌ rn before completion
✅ docs - need for docs assessed and merged, or assessed to be not necessary, set to ❌ docs before completion

## 26 May 2022 ✅
9f19e710f28a1111b07208db43a1ac933aa313cf @ebyhr #12537 ✅ rn ✅ docs
d8e99b0dedaa2e7f32a790b1e3169091c3cedd33 @ebyhr #12532 ✅ rn ✅ docs
b71b5f02f2136656554c663c2a330e43fe98d473 @sopel39 #11289 ✅ rn ✅ docs
83cdee38756a70894c3afff4bbf846309a3a7f12 @sopel39 #11289 ✅ rn ✅ docs
b056e2f5e82c6f49b5442e62ed9732b3ea26ee00 @sopel39 #11289 ✅ rn ✅ docs
e3749a38eaf34e346eb8b568fc61b7550c1a2569 @sopel39 #11289 ✅ rn ✅ docs
65dd2af8e0760bf27d1de92a9487199bdc2c6bf2 @sopel39 #11289 ✅ rn ✅ docs
8e6ff079140206d63bee3ecd4c7830cae104da92 @sopel39 #11289 ✅ rn ✅ docs
46a36ad212887732e49649fe1edc2edfc927c23f @sopel39 #11289 ✅ rn ✅ docs
a6e940127f68ab67e409a7b62c9ed25aad2e663e @losipiuk #12540 ✅ rn ✅ docs
1da52b3a964f726e3323077965dcb8d8ff1376a3 @kokosing #12541 ✅ rn ✅ docs
34c97637a042e27687df0df6996e1da2eb974f54 @kokosing #12541 ✅ rn ✅ docs

## 27 May 2022 ✅ 
a37828bdf096e2de8b56427181b5f51608baa243 @ebyhr #12452 ✅ rn ✅ docs
429464ce90a6f0bc905f31a2bf6cfc89d9b8351d @ebyhr #12452 ✅ rn ✅ docs
2db4430ae9033d6f7d2d6afd24d8f80936d6b200 @ebyhr #12452 ✅ rn ✅ docs
67e5478487b4e060d443dbb31a1fcd4e2a04b170 @ebyhr #12452 ✅ rn ✅ docs
bb0d81448b68b485916a86f9f577cd0f7b55a23b @ebyhr #12452 ✅ rn ✅ docs
0bff849a1786e4aa1dde2bd031f94ba83bac3d57 @findepi #12563 ✅ rn ✅ docs
beba0b8e0f3ac2b9fd7dd2fc2d79175dda0ee3dd @sopel39 #12101 ✅ rn ✅ docs
24442b01d7ab5a663f6046e5a64eb9b7a9852d7e @sopel39 #12101 ✅ rn ✅ docs
111c917ede60b8073964dc5014bc49f5083ca17a @sopel39 #12101 ✅ rn ✅ docs
a262611f9e5df394b1ba5e026c3db08572e6f59d @sopel39 #12101 ✅ rn ✅ docs
15b186d02d4073f4a89e9fd41bb92c4243515760 @sopel39 #12336 ✅ rn ✅ docs
aa26c96f45256d2822ac410ca48f2f8a84d0f889 @sopel39 #12336 ✅ rn ✅ docs
1cf9bc66ee9967cb8d94c948d5c52514b2f6e7cb @sopel39 #12336 ✅ rn ✅ docs
6c5eb59250b0db0ba880c7d9d9cf7d296b1530dd @sopel39 #12336 ✅ rn ✅ docs
629cf5e42a63a62063566eb28d086fda5575f409 @sopel39 #12336 ✅ rn ✅ docs
ad617bcd7fbe6faa485bed3c5688c9e527544a0d @losipiuk #12536 ✅ rn ✅ docs
7e566b9ea078645cea88ed1158222cd2690fdc50 @losipiuk #12536 ✅ rn ✅ docs
11a148f5a56ea53dc9789ac11d1f82c3cbb9f434 @ebyhr #12531 ✅ rn ✅ docs
5a0dcb979ef9c43ee6f4f539068620507f18404a @kasiafi #9831 ✅ rn ✅ docs
73c461790e4a74f475e4795b21c78b00c2994309 @kasiafi #9831 ✅ rn ✅ docs
2feacd6ed06e267f251935174b14b664c9cb66be @kasiafi #9831 ✅ rn ✅ docs
811fb3a6e24a5492c2009fc6dfaaf0fd54730b1a @kasiafi #9831 ✅ rn ✅ docs
dce2a6e9b4de0023111a26a3e3aa8724071dd669 @kasiafi #9831 ✅ rn ✅ docs
e970629d3fff8544fbe78e28a514120f6d822a50 @kasiafi #9831 ✅ rn ✅ docs
14c647e19c83f3e52f0b453c31a3881c586345e9 @kasiafi #9831 ✅ rn ✅ docs
df4edddb78a2bbbefed8cc8ba3333be49748d831 @kasiafi #9831 ✅ rn ✅ docs
9376c367632f5b2f500f927791fa48b61dca9a0b @kasiafi #9831 ✅ rn ✅ docs
a34fc81ee10ec2c8f5f4d8aba7db2b5a629b8e5d @martint #12507 ✅ rn ✅ docs
85650c381860b62fae0a0ea78bd4cbc74089c06c @martint #12507 ✅ rn ✅ docs
61d6268db8bc4ab8b57c21df2cc179a3b1d07f0a @dain #11477 ✅ rn ✅ docs
25424a81dcfa68d162d164551d4ac0dfbae88547 @dain #11477 ✅ rn ✅ docs
760a54b486688d61a74bcbadbfaacf160fb0e554 @dain #11477 ✅ rn ✅ docs
8f709f9ff2479ba6c7013bbc6d377b17a1e3deaf @dain #11477 ✅ rn ✅ docs
a5cdc4d1495e1c0ff52d6ad7f630126bd387986f @dain #11477 ✅ rn ✅ docs
5f907891adddd1afd5603be5e930100444f9962c @dain #11477 ✅ rn ✅ docs
fbf075ef47782bd97bae9b1b9f55ff054758d14d @dain #11477 ✅ rn ✅ docs
ff24f6afabe2e882eb8f3c78fa72667c67226c90 @dain #11477 ✅ rn ✅ docs
b343bba66bb1c85c31e6ede54f889206820867b9 @dain #11477 ✅ rn ✅ docs
d358c57cf63b94616924e27b4c415ec90e5e0106 @dain #11477 ✅ rn ✅ docs
15511fe0ab0d661e7184c6a64a2c857c7ff17221 @dain #11477 ✅ rn ✅ docs
bdbfc01285623644fa1661d71fbeb9b40241aca7 @dain #11477 ✅ rn ✅ docs
b8d6287e16f6c3c1244dbeee135dfb9c98e4a32c @dain #11477 ✅ rn ✅ docs
cb8c031092cd02fde8842679e3e750b1ee97aeb9 @dain #11477 ✅ rn ✅ docs
1ca73834851f270956c6ddb79a2b3a77b80ca9e8 @dain #11477 ✅ rn ✅ docs
39dfb2324338ce7781e01f9f078549cefd5d7c72 @dain #11477 ✅ rn ✅ docs
b94d304fd751f1a17635f303a44ead82c6d5b5bb @dain #11477 ✅ rn ✅ docs
62fecd0dbfdea21609b101f4d2a218e036c79234 @dain #11477 ✅ rn ✅ docs
78bae25755f3be54a4f229d43947aa83559747d3 @dain #11477 ✅ rn ✅ docs
79ef11ca04b1b338c4649e2cef018cd0da8826b9 @dain #11477 ✅ rn ✅ docs
9d85654835d69eb51fe8b9f2887852c8959185f4 @dain #11477 ✅ rn ✅ docs
dc516cfc60d303d864bca90abec128a7e8e14e66 @dain #11477 ✅ rn ✅ docs
dc454015d7ac654fa44907d3877f41382748e9d2 @dain #11477 ✅ rn ✅ docs
d63db3bebfe57ac199eb82cc19cb0193026808a2 @dain #11477 ✅ rn ✅ docs
230cf45053c8f1d59e13a18090199dfa0c748256 @dain #11477 ✅ rn ✅ docs
5a76c238530409aacc42b7be8e886e543d453daa @dain #11477 ✅ rn ✅ docs
2b9734daecf120ef949eee05a4122ac78545499b @dain #11477 ✅ rn ✅ docs
7f9057e9f751e066d7c11535ed67af692d74ce93 @kasiafi #10655 ✅ rn ✅ docs
49113129bd2fd6eef714e0f0b63b0b79df899536 @ebyhr #12574 ✅ rn ✅ docs
9b33e05f0c942c7a319df0eed8a3bd70f7337157 @ebyhr #12490 ✅ rn ✅ docs

## 28 May 2022 ✅ 
cf29082c8735f7d0a994c1867b1525720f7fe279 @findepi #12506 ✅ rn ✅ docs

## 30 May 2022 ✅ 
47dc6d85731cab536b32979b6a49f9870c04d923 @findepi #12558 ✅ rn ✅ docs
5aa7295979225fb01b1c83983064e26ce0377924 @kasiafi #12582 ✅ rn ✅ docs
bd0e07a427884f444fc8dbb4273b2a54a419eecf @ebyhr #12590 ✅ rn ✅ docs
7ebdd44742763f3c16a66c28c77e6c5280ae760d @electrum #12365 ✅ rn ✅ docs
f023b295b4831bf3275004ce7b0c59aa8fbaa874 @electrum #12365 ✅ rn ✅ docs

## 31 May 2022 ❌ 
d9bb461b0c2a82722c6cb2ea2cf6b7cf3ce78653 @sopel39 #12597 ✅ rn ✅ docs
68e32d1e9c6470ff3f42b8dbc7479993ce186871 @kasiafi #12584 ✅ rn ✅ docs
aff1d643c445b4d0c5200b8dcefba27d0e3b963b @losipiuk #12596 ✅ rn ✅ docs
954125e756d79c4bd434c86250fe9bac9ec451f4 @losipiuk #12596 ✅ rn ✅ docs
c49be84c850d1369804d912cf0321dbb68bff962 @losipiuk #12594 ✅ rn ✅ docs
d4ea782f7a53d250dfa6bce9a1b37d8acec91ec4 @ebyhr #12610 ✅ rn ✅ docs
5771df61be1da500e6bfeecde89e280c2a00fd14 @findepi #12609 ✅ rn ✅ docs
bbe9441e823f86536830cc30acc854e3a3ef41e1 @sopel39 #12613 ✅ rn ✅ docs
7f2eeebbda5a175de0509519db758e11eea5950f @arhimondr #12511 ✅ rn ✅ docs
2df44454c699bd59dd12982d045172249960b2e9 @arhimondr #12511 ✅ rn ✅ docs
97dce6adbb69ab3a65df434596a1f9a1cef7f80b @arhimondr #12511 ✅ rn ✅ docs
3595a4f4ce3ca6bb2552f0e18b43e1cb07ce9a9b @findepi #12603 ✅ rn ✅ docs
1a2475da49b2be9a5936730b96a127e1657c252b @kokosing #12544 ✅ rn ❌ docs
08031db4190eb273b07dbf631d1b77322790bc07 @kokosing #12544 ✅ rn ❌ docs
547d11321a1d16167d01f55b115fd4caf1a33e2b @kokosing #12544 ✅ rn ❌ docs
12dda6262a9e17265c49b55900abb01e49cfd315 @kokosing #12544 ✅ rn ❌ docs

## 1 June 2022
2d372010adccd6f2c4b72de56dcaf39fa3a4a81f @ebyhr #12619 ✅ rn ✅ docs
e23e1c40fbf43dd825112a94479b3ec86a9d35f1 @findepi #12577 ✅ rn ✅ docs
011993ff6568c6b47be36fcc0677e174064dd2be @martint #12641 ✅ rn ✅ docs
84162568f31493137e88e0359c14cfcba50c783b @martint #12632 ✅ rn ✅ docs
a4b635babf02632a3f43eccdd5719be86ad4832e @martint #12578 ✅ rn ✅ docs
20daf4e12dedfaafa007b8676a56b69063da963e @martint #12578 ✅ rn ✅ docs
48dab898702ab2fff6c6afbdc5f88373298bade4 @martint #12578 ✅ rn ✅ docs
73488413b28889628bd813d50e5f45addf2b98bd @martint #12620 ✅ rn ✅ docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-383.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Release 383 (1 June 2022)
+
+## General
+
+* Improve query performance by reducing worker-to-worker communication overhead. ({issue}`11289`)
+* Improve performance and reduce memory usage of queries that contain aggregation. ({issue}`12336`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:6]: aggregations.

File: docs/src/main/sphinx/release/release-383.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+* Add IAM Role support for exchange spooling on S3. ({issue}`12444`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:7]: AWS IAM role support

File: docs/src/main/sphinx/release/release-383.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+* Deny `CREATE TABLE` and `CREATE TABLE AS SELECT` statements if a table comment is specified. ({issue}`12452`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:23]: why would that be .. because the connector does not support table comments ? Is there a useful error message?

File: docs/src/main/sphinx/release/release-383.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+## Cassandra connector
+
+* Deny `CREATE TABLE` and `CREATE TABLE AS SELECT` statements if a table comment is specified. ({issue}`12452`)
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Support storing a table comment when creating new tables. ({issue}`12452`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:41]: Make that consistent wth the above ones about using CREATE TABLE  ... maybe by rewording the above entries

File: docs/src/main/sphinx/release/release-383.md (hunk 3 above)
Comment (ebyhr) [reply docs/src/main/sphinx/release/release-383.md:23]: > because the connector does not support table comments ?

Yes

>  Is there a useful error message?

It throws "This connector does not support creating tables with table comment" after this change.

Comment (mosabua) [reply docs/src/main/sphinx/release/release-383.md:23]: Also ... its not so much of a denial.. more a fail ..

Fail table creation when table comments are specified, because table comments are not supported by the connector.

maybe...

Comment (findepi) [reply docs/src/main/sphinx/release/release-383.md:23]: Add something like "Previously, the comment was ignored."

File: docs/src/main/sphinx/release/release-383.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
+# Release 383 (1 June 2022)
+
+## General
+
+* Introduce `json_exists`, `json_query`, and `json_value` functions. ({issue}`9081`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:5]: Can we maybe link to the page?

File: docs/src/main/sphinx/release/release-383.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+* Add AWS IAM Role support for exchange spooling on S3. ({issue}`12444`)
+* Improve query performance by reducing worker-to-worker communication overhead. ({issue}`11289`)
+* Improve performance and reduce memory usage of queries that contain aggregations. ({issue}`12336`)
+* Improve performance of correlated queries involving distinct aggregations. ({issue}`12564`)
+
+## Security
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:11]: Remove the redundant headers by tomorrow morning

File: docs/src/main/sphinx/release/release-383.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+## Web UI
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:13]: empty line below title

File: docs/src/main/sphinx/release/release-383.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+## Cassandra connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`)
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Support storing a table comment when creating new tables. ({issue}`12452`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:47]: creating a new table.

Also .. does that include column comments?

File: docs/src/main/sphinx/release/release-383.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+## Iceberg connector
+
+* Enable execution of the `optimize` procedure for v2 tables. ({issue}`12351`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:59]: maybe we should say Iceberg spec v2 ?

Comment (mosabua) [reply docs/src/main/sphinx/release/release-383.md:59]: or something along those lines

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-383.md (hunk 8 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-383.md:47]: I don't think it includes column comments yet. This is for #12452, which was only table comments.

File: docs/src/main/sphinx/release/release-383.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+* Improve performance and reduce memory usage of queries that contain aggregations. ({issue}`12336`)
+* Improve performance of correlated queries involving distinct aggregations. ({issue}`12564`)
+
+## Web UI
+
+* Clarify format of cumulative user memory on query details page.
+
+## Accumulo connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
```
Comment (ebyhr) [inline docs/src/main/sphinx/release/release-383.md:17]: Remove mention for column comment.

File: docs/src/main/sphinx/release/release-383.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+  Previously, the comment was ignored. ({issue}`12452`)
+
+## Cassandra connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`)
+
+## Delta Lake connector
+
+* Support storing a table comment when creating a new table. ({issue}`12452`)
```
Comment (ebyhr) [inline docs/src/main/sphinx/release/release-383.md:32]: Add 
```
Fail creating a new table if a column comment is specified.
  Previously, the comment was ignored. ({issue}`12574`)
```

**Review by ebyhr — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/release/release-383.md [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+* Fix incorrect `table already exists` error caused by a client timeout when
+  creating a new table. ({issue}`12300`)
+* Fail creating a new table if a column comment is specified. Previously, the
+  comment was ignored. ({issue}`12574`)
+
+## Iceberg connector
+
+* Enable execution of the `optimize` procedure for Iceberg spec v2 tables. ({issue}`12351`)
+* Remove dependency on HiveConfig, which renames `hive.target-max-file-size` to `iceberg.target-max-file-size`
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-383.md:51]: Do we need to talk about dependency and HiveConfig ? I think not..

File: docs/src/main/sphinx/release/release-383.md [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+
+* Introduce `json_exists`, `json_query`, and `json_value` [JSON functions](/functions/json). ({issue}`9081`)
+* Add AWS IAM role support for exchange spooling on S3. ({issue}`12444`)
+* Improve query performance by reducing worker-to-worker communication overhead. ({issue}`11289`)
+* Improve performance and reduce memory usage of queries that contain aggregations. ({issue}`12336`)
+* Improve performance of correlated queries involving distinct aggregations. ({issue}`12564`)
+
+## Security
+
+* Add access control for functions based on function kind.
```
[the remark this answers is not in the document]
Comment (martint) [reply docs/src/main/sphinx/release/release-383.md:13]: This should probably go in the SPI section. It should say something like:

> Allow limiting access to functions based on whether they are scalar, aggregation, window or table functions.

Also, the link to the corresponding issue/PR is missing (issue 12544)

File: docs/src/main/sphinx/release/release-383.md [hunk 14]
```diff
@@ -0,0 +1,125 @@
...
+
+## Web UI
+
+* Clarify format of cumulative user memory on query details page.
```
Comment (martint) [inline docs/src/main/sphinx/release/release-383.md:17]: Missing issue/PR link

File: docs/src/main/sphinx/release/release-383.md [hunk 15]
```diff
@@ -0,0 +1,125 @@
...
+* Fix incorrect query results for certain aggregation queries where aggregation
+  is pushed down  to be executed by the remote database, but the aggregation
+  function result is not needed to evaluate the query. ({issue}`12598`)
+
+## Delta Lake connector
+
+* Support storing a table comment when creating a new table. ({issue}`12452`)
+* Fix incorrect `table already exists` error caused by a client timeout when
+  creating a new table. ({issue}`12300`)
+* Fail creating a new table if a column comment is specified. Previously, the
```
Comment (martint) [inline docs/src/main/sphinx/release/release-383.md:45]: This seems to contradict the entry above:

> Support storing a table comment when creating a new table.

File: docs/src/main/sphinx/release/release-383.md [hunk 16]
```diff
@@ -0,0 +1,125 @@
...
+
+## ClickHouse connector
+
+* Fix incorrect query results for certain aggregation queries where aggregation
+  is pushed down  to be executed by the remote database, but the aggregation
+  function result is not needed to evaluate the query. ({issue}`12598`)
+
+## Delta Lake connector
+
+* Support storing a table comment when creating a new table. ({issue}`12452`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-383.md:42]: > Add support for table comments during table creation.

File: docs/src/main/sphinx/release/release-383.md [hunk 17]
```diff
@@ -0,0 +1,125 @@
...
+* Fix incorrect `table already exists` error caused by a client timeout when
+  creating a new table. ({issue}`12300`)
+* Fail creating a new table if a column comment is specified. Previously, the
+  comment was ignored. ({issue}`12574`)
+
+## Iceberg connector
+
+* Enable execution of the `optimize` procedure for Iceberg spec v2 tables. ({issue}`12351`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-383.md:50]: > Add support for v2 tables to the `optimize` table procedure.

File: docs/src/main/sphinx/release/release-383.md [hunk 18]
```diff
@@ -0,0 +1,125 @@
...
+
+* Fix incorrect query results for certain aggregation queries where aggregation
+  is pushed down  to be executed by the remote database, but the aggregation
+  function result is not needed to evaluate the query. ({issue}`12598`)
+* Fail creating a new table if a column comment is specified. Previously, the
+  comment was ignored. ({issue}`12574`)
+
+## Memory connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
```
Comment (martint) [inline docs/src/main/sphinx/release/release-383.md:69]: Wrong issue? 12452 is about Delta Lake, not memory connector, according to its title.

File: docs/src/main/sphinx/release/release-383.md [hunk 19]
```diff
@@ -0,0 +1,125 @@
...
+  `hive.delete-schema-locations-fallback` to `iceberg.delete-schema-locations-fallback`. ({issue}`12330`)
+
+## Kudu connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`, {issue}`12574`)
+
+## MariaDB connector
+
+* Fix incorrect query results for certain aggregation queries where aggregation
```
Comment (martint) [inline docs/src/main/sphinx/release/release-383.md:61]: > Fix incorrect results for certain queries involving aggregations that are pushed down to the underlying database.

File: docs/src/main/sphinx/release/release-383.md (hunk 15 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-383.md:45]: We support creating a table w/table comments, but it now fails for creating a table w/column comments.

File: docs/src/main/sphinx/release/release-383.md (hunk 18 above)
Comment (colebow) [reply docs/src/main/sphinx/release/release-383.md:69]: I don't think so - 71dcb669621bce7440af26dd94bfcd780d2c680b included this change, and was a part of that PR.

File: docs/src/main/sphinx/release/release-383.md (hunk 15 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-383.md:45]: Ah, sorry, I overlooked the "column" qualifier in the sentence.

**Review by mosabua — APPROVED** [review]
A bit wordy on some entries, but essentially good to go from my perspective.

File: docs/src/main/sphinx/release/release-383.md [hunk 20]
```diff
@@ -0,0 +1,125 @@
...
+  Previously, the comment was ignored. ({issue}`12452`, {issue}`12574`)
+
+## Cassandra connector
+
+* Fail creating a new table if a table comment or a column comment is specified.
+  Previously, the comment was ignored. ({issue}`12452`, {issue}`12574`)
+
+## ClickHouse connector
+
+* Fix incorrect query results for certain aggregation queries where aggregation
```
Comment (martint) [inline docs/src/main/sphinx/release/release-383.md:32]: > Fix incorrect results for certain aggregation queries when aggregations are pushed down to the underlying database.
